Apromore-Custom-Plugins/CSVImporter-Logic/src/main/java/org/apromore/service/csvimporter/io/ParquetLocalFileReader.java

```diff
@@ -51,9 +51,9 @@ public ParquetLocalFileReader(Configuration conf, File file) throws IOException
             try {
                 thread.setContextClassLoader(Path.class.getClassLoader());
                 InputFile inputFile = HadoopInputFile.fromPath(new Path(file.toURI()), conf);
-                ParquetFileReader parquetFileReader = ParquetFileReader.open(inputFile);
-                schema = parquetFileReader.open(inputFile).getFooter().getFileMetaData().getSchema();
-                parquetFileReader.close();
+                try (ParquetFileReader parquetFileReader = ParquetFileReader.open(inputFile)) {
+                    schema = parquetFileReader.getFooter().getFileMetaData().getSchema();
```

Comment:
Everything else in this PR is fairly pure refactoring and doesn't change the original behavior, except for this line. I'm pretty sure it wasn't intentional to call `ParquetFileReader.open` twice.